sdk/storage/azure-storage-blob/tests/test_block_blob_sync_copy.py

```diff
@@ -1,35 +1,33 @@
-# coding: utf-8
-
 # -------------------------------------------------------------------------
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License. See License.txt in the project root for
 # license information.
 # --------------------------------------------------------------------------
 import pytest
```

Comment:
nit: `pytest` goes down above `azure.core.exceptions` since it's a library and not a built-in. Simlarly, for async.

Reply:
Good catch! The `py` in pytest always gets me..